INFO:__main__:PEX 03 start.
INFO:__main__:Connecting to device...
INFO:autopilot:ArduCopter V4.1.0-dev (26a3a02c)
INFO:autopilot:C22MKUTT
INFO:autopilot:Frame: QUAD
INFO:__main__:Device connected.
INFO:__main__:Looking for mission to execute...
INFO:__main__:Arming device...
INFO:__main__:Switching to GUIDED mode...
INFO:__main__:Waiting for arm...
INFO:autopilot:Arming motors
INFO:__main__:Device armed: True.
INFO:__main__:Device takeoff...
INFO:__main__:Current altitude: 0.009
INFO:__main__:Current altitude: 0.009
INFO:__main__:Current altitude: 0.009
INFO:__main__:Current altitude: 0.022
INFO:__main__:Current altitude: 0.29
INFO:__main__:Current altitude: 0.984
INFO:__main__:Current altitude: 1.842
INFO:__main__:Current altitude: 2.832
INFO:autopilot:EKF2 IMU0 in-flight yaw alignment complete
INFO:autopilot:EKF2 IMU1 in-flight yaw alignment complete
INFO:__main__:Current altitude: 3.893
INFO:__main__:Current altitude: 5.007
INFO:__main__:Current altitude: 6.141
INFO:__main__:Current altitude: 7.301
INFO:__main__:Current altitude: 8.477
INFO:__main__:Current altitude: 9.675
INFO:__main__:Current altitude: 10.873
INFO:__main__:Current altitude: 12.084
INFO:__main__:Current altitude: 13.299
INFO:__main__:Current altitude: 14.529
INFO:__main__:Current altitude: 15.754
INFO:__main__:Current altitude: 16.936
INFO:__main__:Current altitude: 17.989
INFO:__main__:Current altitude: 18.803
INFO:__main__:Current altitude: 19.323
INFO:__main__:Changing device mode from VehicleMode:GUIDED to AUTO...
INFO:autopilot:Mission: 1 Takeoff
INFO:autopilot:Mission: 2 WP
INFO:__main__:Device mode = VehicleMode:AUTO.
INFO:__main__:backing up old images...
INFO:root:Searching for target...
INFO:root:Starting camera feed...
INFO:root:configuring rgb stream.
INFO:root:Starting camera streams...
INFO:root:(Potential) target acquired @((0, 0)) with radius 10.
INFO:root:Locking in on lat 21.3219968, lon -157.923997, alt 19.609, heading 195.
INFO:__main__:Program exception: ['Traceback (most recent call last):\n', '  File "C:/Users/C22Mason.Utt/Documents/GitHub/PEX3-COMP495/mission_starter_code.py", line 442, in <module>\n    conduct_mission()\n', '  File "C:/Users/C22Mason.Utt/Documents/GitHub/PEX3-COMP495/mission_starter_code.py", line 329, in conduct_mission\n    cv2.rectangle(frame, (x,y), (x+radius, y+radius))\n', 'TypeError: function takes exactly 4 arguments (2 given)\n']
